assets/js/components/notifications/FirstPartyModeSetupBanner.js

```diff
@@ -63,24 +59,23 @@ export default function FirstPartyModeSetupBanner( { id, Notification } ) {
 	const { setFirstPartyModeEnabled, saveFirstPartyModeSettings } =
 		useDispatch( CORE_SITE );
 
-	const showTooltip = useShowTooltip( id );
-
-	const { isTooltipVisible } = useTooltipState( id );
+	const tooltipSettings = {
+		tooltipSlug: id,
+		title: '',
```

Comment:
Let's not get in the habit of settings empty/pointless values to appease prop types/type checking. If empty values are allowed here, we should allow them and not force empty string as a value.